onnx/helper.py

```diff
@@ -658,6 +658,15 @@ def get_attribute_value(attr: AttributeProto) -> Any:
     raise ValueError(f"Unsupported ONNX attribute: {attr}")
 
 
+def get_node_attr_value(node: NodeProto, attr_name: str) -> Any:
+    matching = [x for x in node.attribute if x.name == attr_name]
+    if len(matching) > 1:
+        raise ValueError(f"Node has multiple attributes with name {attr_name}")
+    if len(matching) < 1:
```

Comment:
```suggestion
    if not matching:
```
tiny nit

Reply:
If this was the only condition, I would agree. But when we handle the 3 conditions (len > 1, = 1, < 1), it is better to be explicit. So, the symmetry with line 663 makes this better, IMHO.